src/main/java/com/github/krzysiekjodlowski/ox/Players.java

```diff
@@ -0,0 +1,61 @@
+package com.github.krzysiekjodlowski.ox;
+
+import com.github.krzysiekjodlowski.ox.model.Symbol;
+
+/**
+ * Container for two players, used
+ * mainly for managing their turns.
+ *
+ * @author Krzysztof Jodlowski
+ */
+class Players implements Subscriber {
+  private final Player playerOne;
+  private final Player playerTwo;
+  private Player currentPlayer;
+
+  private Players(Builder builder) {
+    this.playerOne = builder.playerOne;
+    this.playerTwo = builder.playerTwo;
+    this.currentPlayer = builder.startingPlayer;
+  }
+
+  Player getCurrentPlayer() {
+    return this.currentPlayer;
+  }
+
+  /**
+   * Changes currentPlayer on each move made by another Player.
+   *
+   * @param event represents move made by another Player
+   */
+  @Override
+  public void handle(Event<?> event) {
+    this.currentPlayer = this.currentPlayer.equals(this.playerOne)
+        ? this.playerTwo : this.playerOne;
+  }
```

Comment:
I found no usage of this method. Maybe you should not to implement Subscriber for that class.

Reply:
There must be logic of changing player somewhere. As Players keep both players I found it natural change the current player there. But I will remove Subscriber implementation and provide nextPlayer method instead:)